source/os_specific/service_layers/oszephyr.c

```diff
@@ -967,10 +967,9 @@ UINT64
 AcpiOsGetTimer (
     void)
 {
-    return (k_cycle_get_64 ());
+    return acpi_timer_get();
```

Comment:
Where is `acpi_timer_get()` defined? I can't find it anywhere (neither in the Zephyr tree nor in the acpica tree).

Reply:
> Where is `acpi_timer_get()` defined? I can't find it anywhere (neither in the Zephyr tree nor in the acpica tree).


Also, the commit message talks about tsc timer and pet timer, but this is inconsistent with the content of the commit. I also agree with Tomasz that we shouldn't make assumptions about which timer implementation is available in Zephyr (or what architecture Zephyr is using)

Reply:
> Where is `acpi_timer_get()` defined? I can't find it anywhere (neither in the Zephyr tree nor in the acpica tree).

This Zephyr PR is : https://github.com/zephyrproject-rtos/zephyr/pull/64570

Reply:
> > Where is `acpi_timer_get()` defined? I can't find it anywhere (neither in the Zephyr tree nor in the acpica tree).
> 
> Also, the commit message talks about tsc timer and pet timer, but this is inconsistent with the content of the commit. I also agree with Tomasz that we shouldn't make assumptions about which timer implementation is available in Zephyr (or what architecture Zephyr is using)

acpi_timer_get() is included via acpi_osal.h (which expected to implement per Arch) and currently it enabled/implemented only for x86 (zephyr/acpi/x86_acpi_osal.h)

Updated commit message/heading.